src/utils.ts

```diff
@@ -8,6 +8,11 @@ export const EXIT_CODES = {
   SUCCESS: 0,
   GENERIC_ERROR: 1
 }
+export function isDebugEnabled(): boolean {
+  return (
+    process.env.ENTANDO_CLI_DEBUG === 'true' ?? false
```

Comment:
Yes, we talked about that with @antromeo , in this situation it's better to add a fallback value using the Nullish coalescing "??" Operator

Reply:
I think so, ENTANDO_CLI_DEBUG is a string, and that given code does "if ENTANDO_CLI_DEBUG value is "true" return true or if it is something else or null return false".

It could be 
`return process.env.ENTANDO_CLI_DEBUG === "true" ? true : false` 
or 
```
if(process.env.ENTANDO_CLI_DEBUG === "true") {
    return true 
} else {
    return false
}
```

The current code seems to be more concised.

Reply:
as @avdev4j said , `return process.env.ENTANDO_CLI_DEBUG === "true" ? true : false` it looks nicer to me